packages/sku/lib/certificate.js

```diff
@@ -1,13 +1,13 @@
 const selfsigned = require('selfsigned');
 const { blue } = require('chalk');
 const {
-  existsSync,
+  access: exists,
```

Comment:
I don't think we should rename the `access` import to `exists` since the two operations have different ways of reporting success or failure (i.e. throw an exception vs return a boolean).

If we wanted to keep using the term `exists` in the code we could create a new function to wrap `access`, i.e.:

```typescript
const exists = async (path: string) => {
  try {
    await access(path);

    return true;
  } catch {
    return false;
  }
};
```

Alternatively, we should just use `access` as-is.

Reply:
That's a good point. I'm happy to remove the rename and just use `access` as-is.